tests/e2e/index.js

```diff
@@ -126,6 +127,20 @@ class WriterProcessPool {
 		await fs.rm(`./runtime/${id}`, { recursive: true });
 	}
 
+
+}
+async function copyDir(src,dest) {
```

Comment:
is there a reason to not use `fs.cp(source, destination, { recursive: true }` : https://nodejs.org/api/fs.html#fspromisescpsrc-dest-options ?

Reply:
Ha shit, I forgot this. I took the first answer on Stackoverflow 🤦  I'll update, well spotted

Reply:
[Fixed](https://github.com/writer/writer-framework/compare/da05cf2ef22daf42973e8400bf12413ccef649e8..14578ca3db2259ddf7aad1c4fc7f580a18fb82aa)